cmd/backup/main.go

```diff
@@ -66,14 +66,20 @@ func main() {
 		panic("clusterName not set")
 	}
 
-	var cs api.ClusterSpec
-	sps := os.Getenv(env.ClusterSpec)
-	if err := json.Unmarshal([]byte(sps), &cs); err != nil {
-		logrus.Fatalf("fail to parse backup policy (%s): %v", sps, err)
+	bp, tls, err := parseSpecsFromEnv()
```

Comment:
With this PR, I actually feel like `serveBackupOnly` doesn't need to be a flag.
Can you remove that flag and make it:
```
serveBackupOnly = len(os.Getenv(env.BackupSpec)) != 0
```

Reply:
done